Automattic/jetpack
# Remove the old UI in `/tools.php` for site verification

#### Changes proposed in this Pull Request:

This PR removes the "Site Verification Services" card in Tools as it's a duplicate of the site verification card in `/wp-admin/admin.php?page=jetpack#/traffic`. Morevoer it seems this former version was not maintained as well as the latter, as it only accepts alpha codes.

#### Testing instructions:

- Apply this patch
- Connect to wp-admin on a jetpack site
- Go to `Tools` ( `/wp-admin/tools.php` ) and notice the screen to verify your site on search engines (titled `Website Verification Services (?)`) has been replaced by a placeholder.

Old UI:

![image](https://user-images.githubusercontent.com/230230/44407221-d8a47c00-a55d-11e8-9e60-f8dad7e7daec.png)

New UI:

![image](https://user-images.githubusercontent.com/51896/44542314-45845700-a6c1-11e8-8a02-996bb28b4ff6.png)

- Go to Jetpack > Settings > Traffic, scroll down and notice the `Website Verification Services` card is still available and works as expected

#### Proposed changelog entry:
- Remove the outdated "Site Verification Services" card in Tools
- Removed `jetpack_enable_site_verification` filter. We recommend filtering access to verification tools using `jetpack_get_available_modules` ([docs](https://github.com/Automattic/jetpack/blob/master/docs/partners/managing-modules.md))
